buildLogic/multiplatform-convention/src/main/kotlin/app/cash/sqldelight/multiplatform/MultiplatformConventions.kt

```diff
@@ -53,6 +53,11 @@ class MultiplatformConventions : Plugin<Project> {
 
       jvm()
 
+      js {
+        nodejs()
+        browser()
```

Comment:
Why specify both? It doesn't change ones ability to use in either location.

Reply:
Because I always got these error until I added this: `Caused by: java.lang.IllegalStateException: :runtime is not configured for JS usage
[253](https://github.com/cashapp/sqldelight/actions/runs/3176986581/jobs/5176911797#step:5:254)
	at org.jetbrains.kotlin.gradle.targets.js.npm.resolver.KotlinRootNpmResolver.get(KotlinRootNpmResolver.kt:199)`
But maybe the error was solved with another change too, I will check it and try to use `js()` only.

Reply:
Using `js()` results into the same error again, so I would keep `js { nodejs() browser() }`

Reply:
I'm asking why you have both browser and nodejs. Not the whole js target itself.

Reply:
I know, but using `js()` only didn't work. I had to specify the targets (for unknown reasons)

Reply:
Or do you mean, why do I have to specify both targets? Specifying only one does work too, but specifying none didn't.

Reply:
Specified one target: `browser` only